parquet-protobuf/pom.xml

```diff
@@ -32,8 +32,8 @@
   <properties>
     <elephant-bird.version>4.4</elephant-bird.version>
     <protobuf.version>3.5.1</protobuf.version>
+    <protoc.version>3.5.1</protoc.version>
```

Comment:
Why this new property is required next to the existing `protobuf.version` with the same value?

Reply:
Good question. Can't think of any scenario where having two version (one for protobuf dependency and one for protoc plugin) would be beneficial.

Reply:
So, it seems some cleanup might be required.
I think, it is good to use the same version number for the `protobuf-java` version and the protoc binary version. Let's use the one we already have.
We also have the `protoc.path` property which is not required anymore.